plugin/pkg/scheduler/factory/factory.go

```diff
@@ -65,10 +65,14 @@ type ConfigFactory struct {
 
 	scheduledPodPopulator *framework.Controller
 	modeler               scheduler.SystemModeler
+
+	// SchedulerName of a scheduler is used to select
```

Comment:
Slightly better wording would be "used to select which pods will be processed by this scheduler, based on pod's annotation with key SchedulerAnnotationKey"